tex/alg-NT/frobenius.tex

```diff
@@ -24,7 +30,7 @@ \section{Frobenius elements}
 	Assume $K/\QQ$ is Galois with Galois group $G$.
 	Let $p$ be a rational prime unramified in $K$, and $\kp$ a prime above it.
 	There is a \emph{unique} element $\Frob_\kp \in G$
-	with the property that
+	with the property that, for all $\alpha \in \OO_K$,
```

Comment:
For clarity. (actually thinking about it, is "mod 𝔭" well-defined over the whole field K? If K = ℚ[i] then obviously yes but...?)